en_US/advanced/acl-mnesia.md

```diff
@@ -27,18 +27,12 @@ emqx_auth_mnesia
 
 Mnesia ACL uses the username and password of MQTT packet for authentication by default, and can be changed to use the Client ID and password of MQTT packet for authentication in `etc/plugins/emqx_auth_mnesia.conf` :
 
```

Comment:
Delete this part also? I think it's not relevant anymore for 4.3, where each individual rule has a type (clientid or username).

Reply:
thank you . deleted